### PR TITLE
PCG-13199 : Azure region config

### PR DESCRIPTION
**Pull request by iaashishgupta1**

### CODE REVIEW

File: activityLogsApp/UpdateNSGFlows.cs [hunk 1]
```diff
@@ -116,7 +116,9 @@ public static async Task Run([TimerTrigger("0 */3 * * * *")] TimerInfo myTimer,
 						{
 						    string data =  await response.Content.ReadAsStringAsync();
 						    var result = JsonConvert.DeserializeObject<NSGApiResult>(data);
-						   	await enable_flow_logs(result, nwList, token, subs_id, log);
```
Comment (sachinkumar07) [inline activityLogsApp/UpdateNSGFlows.cs:119]: possible NPE

File: activityLogsApp/UpdateNSGFlows.cs [hunk 2]
```diff
@@ -116,7 +116,9 @@ public static async Task Run([TimerTrigger("0 */3 * * * *")] TimerInfo myTimer,
...
+						    string[] networkWatcherRegions = Environment.GetEnvironmentVariable("networkWatcherRegions").Split(',');
```
Comment (sachinkumar07) [inline activityLogsApp/UpdateNSGFlows.cs:119]: possible NPE

Comment (iaashishgupta1) [reply activityLogsApp/UpdateNSGFlows.cs]: fixed

File: activityLogsApp/UpdateNSGFlows.cs [hunk 3]
```diff
@@ -116,7 +116,12 @@ public static async Task Run([TimerTrigger("0 */3 * * * *")] TimerInfo myTimer,
 						{
 						    string data =  await response.Content.ReadAsStringAsync();
 						    var result = JsonConvert.DeserializeObject<NSGApiResult>(data);
-						   	await enable_flow_logs(result, nwList, token, subs_id, log);
+						    string[] networkWatcherRegions;
+						    if(Environment.GetEnvironmentVariable("networkWatcherRegions") != null){
```
Comment (sachinkumar07) [inline activityLogsApp/UpdateNSGFlows.cs:120]: rename it to nwRegions

**Review by sachinkumar07 — APPROVED** [review]

**Pull request merged**